.github/workflows/main.yml

```
@@ -49,8 +49,9 @@ jobs:
         shell:
           - bash
           #- dash # for py 3.12 also test dash as default shell (snakemake will still use bash for shell commands)
-        exclude: []
-          #- shell: dash 
+        exclude:
+          []
```

actionlint (1.7.4)

"exclude" section should not be empty

(syntax-check)